core/PRP/Geometry/plDrawableSpans.cpp

```diff
@@ -35,6 +35,8 @@ plDrawableSpans::~plDrawableSpans()
         delete *group;
     for (auto span = fSpans.begin(); span != fSpans.end(); ++span)
         delete *span;
+    for (auto span = fSourceSpans.begin(); span != fSourceSpans.end(); ++span)
+        delete *span;
```

Comment:
```suggestion
    for (auto span : fSourceSpans)
        delete span
```

Reply:
Yeah, I was going for consistency with the rest of the file here :wink:

Reply:
Hmm, fair point...  But I'm not opposed to modernizing new code even in files with older code ;)

Reply:
And of course the build break is my fault (missing `;`)...  :/